### PR TITLE
Add logging. Fix project naming in gitlab. Handle PKIX issues.

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/git/Git.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ protected void pushRepository(Path path, String httpTransportUrl, String commit,
             Log.infof("Updating current origin of %s to %s", jConfig.getString("remote", "origin", "url"),
                     httpTransportUrl);
             jConfig.setString("remote", "origin", "url", httpTransportUrl);
+            jConfig.setBoolean("http", null, "sslVerify", false);
```
Comment (stuartwdouglas) [inline java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/git/Git.java:64]: We definitely don't want to hard code this, it should be optional for testing/dev.

File: java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/mavenrepository/MavenRepositoryDeployer.java [hunk 2]
```diff
@@ -49,6 +49,8 @@ public MavenRepositoryDeployer(BootstrapMavenContext mvnCtx, String username, St
         this.system = mvnCtx.getRepositorySystem();
         this.session = MavenRepositorySystemUtils.newSession();
 
+        Log.infof("Maven credentials are username '%s', password '%s', and repository '%s'", username, password, repository);
```
Comment (stuartwdouglas) [inline java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/mavenrepository/MavenRepositoryDeployer.java:52]: We should not log the password here.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]

File: java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/git/Git.java (hunk 1 above)
Comment (rnc) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/git/Git.java:64]: I needed this to get the demo with gitlab working.

File: java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/mavenrepository/MavenRepositoryDeployer.java (hunk 2 above)
Comment (rnc) [reply java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/mavenrepository/MavenRepositoryDeployer.java:52]: Yeah agreed, I'll remove it. I needed this logging initially as the `kubectl get secret` wasn't showing the trailing newline. Need to find a better way of determining those sorts of problems (even manually via cmd line)

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**